OneCommunityGlobal/HGNRest
# Jatin improve report loading time

# Description
<img width="775" alt="Screenshot 2024-09-25 at 11 02 20 PM" src="https://github.com/user-attachments/assets/e5d788c6-30cb-42d2-83cb-0f9072fb106e">


## Related PRS (if any):
This backend PR is related to the [#2780](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2780) frontend PR.
…

## Main changes explained:
- Updated src/controllers/timeEntryController.js A for updating query.
- Update file src/models/timeentry.js

## How to test:
1. check into current branch
2. do `npm install` and `npm run build` and `npm start` to run this PR locally
3. Go to frontend Repo for further steps.

## Screenshots or videos of changes:
https://github.com/user-attachments/assets/c1454060-8f0e-41df-a8ae-041d133bc1e3
